src/alchemlyb/estimators/mbar_.py

```diff
@@ -45,6 +45,9 @@ class MBAR(BaseEstimator):
     states_ : list
         Lambda states for which free energy differences were obtained.
 
+    overlap_matrix: DataFrame
```

Comment:
really a DataFrame... I thought that pymbar returns numpy arrays. please check and correct if necessary